### PR TITLE
Add systemd timer and service for updates

### PR DESCRIPTION
**Pull request by kalikiana**
See: https://progress.opensuse.org/issues/166721

### CODE REVIEW

File: systemd/os-autoinst-scripts-update-git.service [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Update git repository of os-autoinst-scripts
+
+[Timer]
+OnCalendar=*:0/3
+Unit=os-autoinst-scripts-update-git.service
```
Comment (baierjan) [inline systemd/os-autoinst-scripts-update-git.service:6]: That's in the `.timer`, you are missing `[Service]` section.

**Review by baierjan — CHANGES_REQUESTED** [review]

File: systemd/os-autoinst-scripts-update-git.timer [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Update git repository of os-autoinst-scripts
+
+[Timer]
+OnCalendar=*:0/3
+Unit=os-autoinst-scripts-update-git.service
+
+[Install]
+WantedBy=default.target
```
Comment (baierjan) [inline systemd/os-autoinst-scripts-update-git.timer:9]: I think the default target should be
```suggestion
[Install]
WantedBy=timers.target
```

Comment (kalikiana) [reply systemd/os-autoinst-scripts-update-git.timer:9]: I've seen both 🤔 Not sure what the difference is in practice. I don't see an explicit mention of that in [systemd timer upstream docs](https://man.archlinux.org/man/systemd.timer.5#OPTIONS) or systemd.unit.

Comment (baierjan) [reply systemd/os-autoinst-scripts-update-git.timer:9]: https://www.freedesktop.org/software/systemd/man/latest/systemd.special.html#timers.target

File: systemd/os-autoinst-scripts-update-git.service [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+Description=Update git repository of os-autoinst-scripts
+After=network-online.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/git -C /opt/os-autoinst-scripts pull --quiet --rebase origin
+User=geekotest
+
+[Install]
+WantedBy=default.target
```
Comment (baierjan) [inline systemd/os-autoinst-scripts-update-git.service:11]: You might want to drop the install section as this service is supposed to be started only with the timer and not actually enabled?

**Review by Martchus — APPROVED** [review]

**Review by baierjan — APPROVED** [review]

**Pull request merged**